### DISCUSSION TITLE
Give the Tree node a more distinct name

### DISCUSSION THREAD

**Discussion by ScottyWired** [godotengine/godot-proposals#6859] [GUI]
Just for searching purposes outside of Godot.

Whether on YouTube or the Discord or anywhere else, it's very hard to find resources about the Tree node due to noise.
The only tutorial I found is five years old, because the results are crowded out with "node tree", "scene tree", "behaviour tree", "animation tree", and just literal leafy trees.

Some unique phrase like "StructuredTree" would help this node stand out and would make documentation more accessible.

**Comment by theraot** [comment]
I believe that people naturally refer to it as "Tree node". Try searching that. *Notice that is how you refer to it on the title of the question. I have answered at least one question on it, and they called it "Tree node" too: [Godot: Extract all data from a tree node](https://stackoverflow.com/questions/74595331/godot-extract-all-data-from-a-tree-node).*

I'm not convinced the suggested name `StructuredTable` is good. If we are going to rename `Tree`, I would rather it be after the fact that more often than not a kind of table. So `TreeTable` or `TableTree`?

By the way, getting to the documentation on it from Godot itself isn't much trouble. I assume it didn't cover what you are trying to figure out. Perhaps you could point to what could be improved about its documentation?

**Reply by ScottyWired** [reply]
That Stack Overflow question you linked is doing almost exactly what I'm trying to do. Searching `"tree node" godot` brings that result up on the third page of results for me, and searching without quotes does not bring it up at all.

My discussion post didn't mention it, but I am indeed trying to make a table. And that's what I've noticed in the documentation for Tree- it never mentions the word 'table'. If that's one of its largest uses, then the word should appear at least once, surely.

I think my issue wasn't "how to make a table with tree node" but rather I spent a long task asking "CAN I make a table with tree node".
The answer was drowned out by other tree-related results.

**Comment by AThousandShips** [comment]
I think breaking compatibility, making names more cumbersome, and possibly more confusing by restricting them to one specific behaviour (like table) is not warranted over it being difficult to search for in forums

**Reply by theraot** [reply]
From the reply to my post, probably an example in the official documentation showing how to use it as a table would suffice.
